samwise.js

```diff
@@ -1,77 +1,77 @@
 /**
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.